docker/database.sql

```diff
@@ -0,0 +1,59 @@
+DROP TABLE categories;
+DROP TABLE product;
+DROP TABLE basket_element;
+DROP TABLE basket;
+
+CREATE TABLE categories (
+    id SERIAL PRIMARY KEY NOT NULL,
+    name VARCHAR (255) NOT NULL,
+    categorie_type VARCHAR (255) NOT NULL
+);
+
+INSERT INTO categories(name, categorie_type) VALUES('Bundle Server', 'server');
+INSERT INTO categories(name, categorie_type) VALUES('Network', 'network');
+
+CREATE TABLE product (
+    id SERIAL PRIMARY KEY NOT NULL,
+    titre TEXT UNIQUE NOT NULL,
+    description TEXT,
+    cost NUMERIC NOT NULL,
+    quantity INTEGER NOT NULL,
+    categorie_id INTEGER NOT NULL
+);
+
+INSERT INTO product(titre, description, quantity, categorie_id, cost) VALUES('Bundle Dell R610', '2 Cpu<br/>16Go de Ram<br/>2 disque de faible capacite (60Go min)', 12, (SELECT id FROM categories WHERE name = 'Bundle Server'), 20.0);
```

Comment:
if you know it won't trigger any conflict, you can chain multiple `('','')` after `VALUES`, ie. 

```sql
INSERT INTO product (title, description, quantity, category_id, cost)
VALUES
  ('some', 'thing', 1, 1, 1),
  ('other', 'thing', 1, 2, 3),
  ('last', 'stuff', 1, 1, 1);
```

Reply:
btw its @hregibo i just forgot to swap account